main.go

```diff
@@ -94,6 +103,22 @@ func main() {
 		c.Start()
 	}
 
+	exporter, err := prometheus.NewExporter(prometheus.Options{})
+	if err != nil {
+		glog.Fatalf("can't create prometheus exporter: %v", err)
+	}
+	view.RegisterExporter(exporter)
+
+	mux := http.NewServeMux()
+	mux.Handle("/metrics", exporter)
+	srv := &http.Server{
+		Addr:    ":9999",
```

Comment:
@enisoc I want to get your opinion on how we should configure this. My initial guess would be to just use an environment variable, but I wanted to discuss.

Reply:
I've added a flag to configure this.